src/internal/pdf/pdf.go

```diff
@@ -0,0 +1,61 @@
+package pdf
+
+import (
+	"errors"
+	"path/filepath"
+	"time"
+
+	"github.com/signintech/gopdf"
+)
+
+const (
+	PAGE_WIDTH  = 162
+	PAGE_HEIGHT = 90
+	PAGE_MARGIN = 10
+)
+
+// Generate a PDF label consisting of the provided `labelText` and the current date
+// returns
+func GeneratePdf(labelText string, filePathName string) error {
+
+	pdf := gopdf.GoPdf{}
+	pdf.Start(gopdf.Config{PageSize: gopdf.Rect{W: PAGE_WIDTH, H: PAGE_HEIGHT}})
+	pdf.AddPage()
+
+	fontFilePath, err := filepath.Abs("./fonts/Rubik-Regular.ttf")
+	if err != nil {
+		return err
+	}
+
+	err = pdf.AddTTFFont("rubik", fontFilePath)
+	if err != nil {
+		return err
+	}
+
+	err = pdf.SetFont("rubik", "", 16)
+	if err != nil {
+		return err
+	}
+
+	pdf.SetXY(PAGE_MARGIN, 10)
+	err = pdf.SetFont("rubik", "", 16)
+	pdf.SetTextColor(0, 0, 0)
+	pdf.Cell(nil, labelText)
+
+	pdf.SetXY(PAGE_MARGIN, 55)
+	err = pdf.SetFont("rubik", "", 10)
+	pdf.SetTextColor(85, 85, 85)
+	pdf.Cell(nil, "made:")
+
+	pdf.SetXY(PAGE_MARGIN, 67)
+	err = pdf.SetFont("rubik", "", 12)
+	pdf.SetTextColor(0, 0, 0)
+	pdf.Cell(nil, time.Now().Local().Format(time.DateOnly))
+
+	err = pdf.WritePdf(filePathName)
+	if err != nil {
+		return errors.New("Unable to generate PDF, an unknown error occurred")
```

Comment:
You are swallowing actual error. I suggest adding it to the error you are throwing

```
errors.New("Unable to generate PDF, an unknown error occurred" + err.Error())
```

or then just write:

```
	if err != nil {
		return err
	}
```